### PR TITLE
pricing paper on front pages & small faq modification

### PR DESCRIPTION
**Pull request by arjunhassard**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [x] 2
- [ ] 3

**What this does:**
Makes resource on network fees more accessible 

**Issues fixed/closed:**
None

**Why it's needed:**
This doesn't address any productdev issues!

**Notes for reviewers:**
None, should be easy to approve!

### CODE REVIEW

**Review by cygnusv — APPROVED** [review]

File: docs/source/support/faq.rst [hunk 1]
```diff
@@ -140,7 +140,7 @@ Inflation Rewards (NU) and Policy Fees (ETH).
 Q: How are Policy Fees (ETH) determined?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The fee is calculated based on each period committed to and the number of policies that the Ursula is enforcing.
+The amount of fees earnable by a staker in a given period is determined by multiple factors, including (1) the minimum fee rate set by the staker, (2) the number of active policies issued by users of the network to that staker, and (3) whether the staker has previously committed to being available for that period. See `Service Fees (Pricing) <https://docs.nucypher.com/en/latest/architecture/service_fees.html>`_ for details.
```
Comment (derekpierre) [inline docs/source/support/faq.rst:143]: @arjunhassard I'm never one to question your vocabulary prowess 😅 - but is "earnable" a word?

**Review by derekpierre — COMMENTED** [review]
Looks good - but needs a newsfragment - see https://github.com/nucypher/nucypher/blob/main/newsfragments/README.md.

TL;DR create a file in `nucypher/newsfragments` named `2520.doc.rst` with a one-liner outlining the crux of the change.

File: docs/source/support/faq.rst (hunk 1 above)
Comment (cygnusv) [reply docs/source/support/faq.rst:143]: "Earnable" sounds good to me, and seems to have at least some marginal use for those that speak legalese, but I guess we can phrase it in a simpler way:
```suggestion
The amount of fees a staker can earn in a given period is determined by multiple factors, including (1) the minimum fee rate set by the staker, (2) the number of active policies issued by users of the network to that staker, and (3) whether the staker has previously committed to being available for that period. See `Service Fees (Pricing) <https://docs.nucypher.com/en/latest/architecture/service_fees.html>`_ for details.
```

Comment (KPrasch) [reply docs/source/support/faq.rst:143]: The FAQ has turned into a nursery for actual documentation.

Comment (arjunhassard) [reply docs/source/support/faq.rst]: @KPrasch Isn't that kind of what an FAQ is for? To give users a quick, simplified answer without having to trudge through the whole spec – thereby addressing high-level confusion before a rage/exhaustion-quit? 

Do you think this change makes it more or less of a nursery? I corrected a misleading answer and added a link to the actual documentation.

Comment (KPrasch) [reply docs/source/support/faq.rst]: I'm not saying it's a bad thing, just an observation.  Even before this PR, the FAQ encompasses too much essential information to be properly composed and discovered here.  I even think it's part of the lifecycle of documentation for our organization, certainly better to have *some* kind of reference in the form of an FAQ than none at all.

**Comment by arjunhassard** [comment]
@derekpierre 
> Looks good - but needs a newsfragment - see https://github.com/nucypher/nucypher/blob/main/newsfragments/README.md.
> 
> TL;DR create a file in `nucypher/newsfragments` named `2520.doc.rst` with a one-liner outlining the crux of the change.

Done, thanks for heads-up!

File: docs/source/support/faq.rst (hunk 1 above)
Comment (arjunhassard) [reply docs/source/support/faq.rst]: @KPrasch agreed, it's particularly intimidating to a fresh developer/user to have to navigate through so much information primarily aimed at stakers. been saying for a long time we need to split all our educational materials by audience (demand-side and service-side) – maybe now's the time?

Comment (cygnusv) [reply docs/source/support/faq.rst]: Our old friend #1001

Comment (derekpierre) [reply docs/source/support/faq.rst]: Some initial work by @KPrasch , #2463 .

**Review by derekpierre — APPROVED** [review]
🎸
